gradle/libs.versions.toml

```diff
@@ -92,9 +93,14 @@ spring-cloud-starter-gateway = { module = "org.springframework.cloud:spring-clou
 spring-cloud-starter-kubernetes-client-config = { module = "org.springframework.cloud:spring-cloud-starter-kubernetes-client-config", version.ref = "spring-cloud-kubernetes" }
 spring-boot-starter-oauth2-client = { module = "org.springframework.boot:spring-boot-starter-oauth2-client" }
 spring-context-indexer = { module = "org.springframework:spring-context-indexer", version.ref = "spring" }
+spring-kafka = {module = "org.springframework.kafka:spring-kafka"}
+spring-kafka-test = {module = "org.springframework.kafka:spring-kafka-test"}
 spring-web = { module = "org.springframework:spring-web", version.ref = "spring" }
 jackson-module-kotlin = { module = "com.fasterxml.jackson.module:jackson-module-kotlin" }
 
+kafka-clients = {module = "org.apache.kafka:kafka-clients", version.ref = "kafka-client"}
```

Comment:
these both versions also presented in [spring-boot-dependencies](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.7.3/spring-boot-dependencies-2.7.3.pom)

Reply:
fixed